reference/tax_provider.yml

```
@@ -1332,6 +1285,42 @@ components:
         - rate
         - amount
       x-internal: false
+    item_type:
+      type: string
+      description: |-
+        The type of item for the line item in the document.
+
+        Tax estimate requests for order-level refunds use the type `refund`.
+      enum:
+        - item
+        - refund
```

```suggestion
        - refund
```
@theromulans  When a request is made for an order-level refund, is `refund` the `type` for all the line items? Is `refund` an allowed value for **each** type of line item (e.g. wrapping line items, handling line items, etc)?